sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTest.java

```diff
@@ -2329,23 +2397,96 @@ public void analyzeLinkedEntityActions(HttpClient httpClient, TextAnalyticsServi
             syncPoller.waitForCompletion();
             PagedIterable<AnalyzeActionsResult> result = syncPoller.getFinalResult();
 
+            validateAnalyzeBatchActionsResultList(false, false,
+                asList(getExpectedAnalyzeBatchActionsResult(
+                    IterableStream.of(Collections.emptyList()),
+                    IterableStream.of(asList(getExpectedRecognizeLinkedEntitiesActionResult(false,
+                        LINKED_ENTITIES_RECOGNITION_NAME, TIME_NOW, getRecognizeLinkedEntitiesResultCollection(), null))),
+                    IterableStream.of(Collections.emptyList()),
+                    IterableStream.of(Collections.emptyList()),
+                    IterableStream.of(Collections.emptyList()))),
+                result.stream().collect(Collectors.toList()));
+        });
+    }
+
+    @Disabled("Linked entity recognition action doesn't contains bingId property. https://github.com/Azure/azure-sdk-for-java/issues/22208")
```

Comment:
have we reported this bug to the service team?

Reply:
Yes